packages/tokens/styles/color.scss

```diff
@@ -0,0 +1,32 @@
+$white: #fff;
+$black: #000;
+
+// Primary colours
+$ca-palette-ink: #3e4543;
+$ca-palette-paper: #f2edde;
+$ca-palette-coral: #f04c5d;
+
+// Secondary colours
+$ca-palette-seedling: #45ad8f;
+$ca-palette-ocean: #1b7688;
+$ca-palette-wisteria: #727193;
+$ca-palette-lapis: #253c64;
+$ca-palette-peach: #f3786d;
+$ca-palette-yuzu: #ffce1e;
+
+// Tertiary colours
+$ca-palette-stone: add-tint($ca-palette-ink, 97%);
```

Comment:
`add-tint` is a mixin that comes from kaizen-component-library/styles/color.scss, not a scss built in. So, in it's current state this file won't compile.

There might be a cyclic dependency issue if kaizen-component-library depends on kaizen-design-tokens which depends on kaizen-component-library (for the mixins). Scss helpers like this might need to be moved into another package to avoid the dependency, or they could be moved here (but then the nice the token vs mixin separation you have going here is lost). A more controversional way to do it is just hardcode hex values for the ones here that use the mixins 🤔 . Perhaps that's not such a terrible thing. We are moving towards having hex values for all tokens anyway.

Reply:
It'd make compilation a few nanoseconds faster :)

Reply:
Really good catch! This is one of the reasons I think it's good to think about where these tools and helpers might belong. Some might best fit in the component library, but others would be better suited to their own special package. 

In this case i'll change this to a hex/rgba value.